dist-scripts/debian/gmrender-resurrect.default

```diff
@@ -25,17 +25,29 @@ INITIAL_VOLUME_DB=-10
 # 'aplay -L'), then gmediarenderer will use that ALSA device to play 
 # audio. Otherwise, whatever default is configured for gstreamer for the 
 # '$DAEMON_USER' is used.
-ALSA_DEVICE="sysdefault"
+RET=`dpkg -l pulseaudio`
```

Comment:
Here, you are making the choice of pulseaudio vs. Alsa dependent on the existence of a particular packet, which sounds like it will result in non-repeatable results: if accidentally a pulseaudio package is installed, suddenly the way gmrender works is changing, which can make things surprising.

So I suggest to rather have this as a variable at the top (USE_PULSEAUDIO=1 or something).